development-docs/RELEASE.md

```diff
@@ -13,44 +13,45 @@ The release process should normally look like this:
    * Copy the build ID (from the URL in Azure Pipelines)
-10. Run the `operators-release` pipeline manually in Azure pipelines
-    * Select the release branch from the list
-    * Set the desired release version (e.g. `0.45.0-rc1` for RCs or `0.45.0` GA releases)
-    * Set the release suffix as `0`
-    * Set the build ID to the build ID from previous step (For GA, this should be the build ID used for the last RC since there should be no changes)
-11. Once the release build is complete:
+9. Run the `operators-release` pipeline manually in Azure pipelines
+   * Select the release branch from the list
+   * Set the desired release version (e.g. `0.45.0-rc1` for RCs or `0.45.0` GA releases)
+   * Set the release suffix as `0`
+   * Set the build ID to the build ID from previous step (For GA, this should be the build ID used for the last RC since there should be no changes)
+   * Set the `CENTRAL_USERNAME` and `CENTRAL_PASSWORD` variables to your Maven Central user token
+10. Once the release build is complete:
     * Download the release artifacts (binary, documentation and sbom)
     * Check the images pushed to Quay.io
     * Mark the build in the Azure Pipelines UI to be retained forever
-12. Create a GitHub tag and release based on the release branch. Attach the release artifacts and docs as downloaded from the Azure pipelines.
+    * Go to Maven Central > Publish and publish the release artifacts
+11. Create a GitHub tag and release based on the release branch. Attach the release artifacts and docs as downloaded from the Azure pipelines.
+    * For GAs, prepare steps 13 and 14 before creating the release in GitHub.
     * For RCs, the tag should be named with the RC suffix, e.g. `0.45.0-rc1`
-13. Run system tests pipelines. For more details see [Running Azure System Tests pipelines](#running-azure-system-tests-pipelines) section.
-14. _(only for GA, not for RCs)_ Update the website
+12. _(only for RCs, not for GAs) Run system tests pipelines. For more details see [Running Azure System Tests pipelines](#running-azure-system-tests-pipelines) section.
+13. _(only for GA, not for RCs)_ Update the website
     * Update the `_redirects` file to make sure the `/install/latest` redirect points to the new release.
     * Update the `_data/releases.yaml` file to add new release
+    * Download the release artifacts from the CI pipeline and unpack them
     * Update the documentation: 
       * Create new directories `docs/operators/<new-version>` and `docs/operators/<new-version>/full` in the website repository
-      * Copy files from the operators repository `documentation/htmlnoheader` to `docs/operators/<new-version>` in the website repository
-      * Copy files from the operators repository `documentation/html` to `docs/operators/<new-version>/full` in the website repository
+      * Delete the old HTML files and images from `docs/operators/latest` and `docs/operators/latest/full` (keep the `*.md` files)
```

Comment:
I used the release artifact here, since it's needed for the helm command it worked well, but correct me if this is wrong